test/scripts/aks-shared-vnet.sh

```diff
@@ -37,7 +37,8 @@ az network vnet subnet create \
     --address-prefixes 10.2.0.0/16
 
 # Create aks member cluster1.
-az aks create \
+if [ "$ENABLE_TRAFFIC_MANAGER" == "false" ]; then
```

Comment:
Can we just always create clusters with BYO identities?

Reply:
would like to keep one test for testing without ATM to make sure without any permission, mcs should work.

Reply:
We can only assign permissions when enabling ATM but always use BYO identities?

Reply:
I think we need to keep no BYO identity clustrers so we know it still works